test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14218,13 +14218,17 @@ def create_subjob_id(self, job_array_id, subjob_index):
         return job_array_id[:idx + 1] + str(subjob_index) + \
             job_array_id[idx + 1:]
 
-    def create_eatcpu_job(self, duration=None):
+    def create_eatcpu_job(self, duration=None, mom=None):
         """
         Create a job that eats cpu indefinitely or for the given
         duration of time
         """
         script_dir = os.path.dirname(os.path.dirname(__file__))
         script_path = os.path.join(script_dir, 'utils', 'jobs', 'eatcpu.py')
+        d = os.path.join(os.path.sep, 'home', self.username)
+        if mom:
+            DshUtils().run_copy(hosts=mom, src=script_path, dest=d)
```

Comment:
you should probably check here if mom is localhost or not, if it's localhost then you don't need to do the remote copy.

Reply:
In some of the text in ptl i see this: pbs_benchpress -p server=remote@/etc/pbs.conf, in this case checking for localhost will not help. I feel checking if server name and mom name are equal is better option here. What do you think?

Reply:
I agree with @agrawalravi90, make use DshUtils.is_localhost on mom and do run_copy only if it is a remote mom.